src/lib/schemas/translateAggregationType.ts

```diff
@@ -0,0 +1,29 @@
+import i18n from '@dhis2/d2-i18n'
+
+const translations: { [key: string]: string } = {
+    SUM: i18n.t('Sum'),
+    AVERAGE: i18n.t('Average'),
```

Comment:
We already have these here https://github.com/dhis2/maintenance-app-beta/blob/master/src/constants/translatedModelConstants.ts

Reply:
I removed my duplicate translations. I can't use your helpers until you merged, so will adjust code once your changes are in master